experiment_buddy/experiment_buddy.py

```diff
@@ -219,7 +217,11 @@ def _ask_experiment_id(cluster, sweep):
         experiment_id = tkinter.simpledialog.askstring(title, "experiment_id")
         root.destroy()
     except:
-        experiment_id = input(f"Running on {title} \ndescribe your experiment (experiment_id):\n")
+        if os.environ['TEST_BRANCH_NAME']:
+            import uuid
```

Comment:
we should use an unique random string across all the process, not sure which tho, so it's ok for now.